checks/libs/win/winpdh.py

```diff
@@ -8,24 +8,24 @@ class WinPDHCounter(object):
     # store the dictionary of pdh counter names
     pdh_counter_dict = {}
 
-    def __init__(self, class_name, instance_name, log):
+    def __init__(self, class_name, counter_name, log, instance_name = None):
```

Comment:
Will making these changes affect the pdh check in integrations core?

Reply:
No; the pdh check always will supply no instance name. The problem is confusion between what PDH calls countersets, counters, and instances, and what we generally refer to as instances.  The generic PDH counter does all instances of the counters, so it's `instance_name` param is always none.